src/core/thread/network_data_local.cpp

```diff
@@ -379,7 +379,7 @@ otError Local::UpdateInconsistentServerData(Coap::ResponseHandler aHandler, void
         mOldRloc = Mac::kShortAddrInvalid;
     }
 
-    SuccessOrExit(error = NetworkData::SendServerDataNotification(mOldRloc, aHandler, aContext));
+    SuccessOrExit(error = SendServerDataNotification(mOldRloc, aHandler, aContext));
```

Comment:
Apply this change here as well? https://github.com/openthread/openthread/blob/42ecab1a2495960c33fa9590ad9111c7034abc19/src/core/thread/network_data_leader_ftd.cpp#L1387

Reply:
Yes. Thank you. 👍